src/operon-runtime/config.ts

```diff
@@ -26,6 +26,7 @@ export interface ConfigFile {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     user_dbclient?: UserDatabaseName;
   };
+  system_database: string;
```

Comment:
It seems to duplicate with the `system_database` option above. And we probably don't want users to control whether we use FDB or not (we force Postgres for free tiers). Can we configure the system DB from the cloud side?